cpp/src/io/parquet/experimental/hybrid_scan_preprocess.cu

```diff
@@ -142,7 +141,7 @@ void hybrid_scan_reader_impl::prepare_row_groups(
     create_global_chunk_info();
 
     // compute schedule of input reads.
-    compute_input_passes(read_mode::READ_ALL);
+    compute_input_passes(mode);
```

Comment:
We will process all input row groups in one pass but multiple subpasses and output table chunks to respect memory limits